kinetic/distribution.yaml

```diff
@@ -2328,6 +2328,30 @@ repositories:
       url: https://github.com/ct2034/dockeROS.git
       version: master
     status: developed
+  doosan-robot:
+    doc:
+      type: git
+      url: https://github.com/doosan-robotics/doosan-robot.git
+      version: master
+    release:
+      packages:
+      - doosan_robotics
+      - dsr_bringup
+      - dsr_control
+      - dsr_description
+      - dsr_example_cpp
+      - dsr_example_py
+      - dsr_gazebo
+      - moveit_config_m0609
+      - moveit_config_m0617
+      - moveit_config_m1013
+      - moveit_config_m1509
+      - robotiq_description
+      tags:
+        release: release/kinetic/{package}/{version}
+      url: https://github.com/doosan-robotics/doosan-robot.git
```

Comment:
This is not a release url. The release section should be populated by the bloom release tool after being run. The release stanza should not be filled in manually.